tests/route/test_default_route.py

```diff
@@ -33,15 +34,15 @@ def get_upstream_neigh(tb, device_neigh_metadata):
     if topo_cfg_facts is None:
         return upstream_neighbors
 
-    for neigh_name, neigh_cfg in list(topo_cfg_facts.items()):
+    for neigh_name, neigh_cfg in topo_cfg_facts.iteritems():
         if neigh_type not in neigh_name:
```

Comment:
@wenyiz2021 why this changes ?

Reply:
I copied wrong change, local branch wasn't in sync